src/subdomains/supporting/payin/entities/crypto-input.entity.ts

```diff
@@ -99,8 +101,8 @@ export class CryptoInput extends IEntity {
   @Column({ default: false })
   isConfirmed: boolean;
 
-  @Column({ length: 256, nullable: true })
-  amlCheck: CheckStatus;
+  @Column({ nullable: true })
+  isForwardConfirmed: boolean;
```

Comment:
Hm, we already have a `isConfirmed`column 🤔 so I would either not change it or change both names because on the one hand `isConfirmed` and on the other hand e.g. `approved` is a bit strange or? 😅